docs/blog/2020-04-29-incredimental-builds/index.md

```diff
@@ -15,7 +15,7 @@ This is a really exciting vision, but it only works if builds are lightning quic
 
 Happily, this vision has gotten significantly more tangible recently, with our [incremental builds launch](https://gatsbyjs.org/blog/2020-04-22-announcing-incremental-builds/) last week 🎉
 
-# What are incremental builds?
+## What are incremental builds?
```

Comment:
Sorry I forgot to do this, @marcysutton 🙈I know you've given me this feedback before. My personal blog maps `#` to `h2`, and it slipped my mind.

Will catch it beforehand next time!